lib/mojito.js

```diff
@@ -220,13 +197,12 @@ Mojito = (function ()
                     }
                 }
 
-                this.options.newToThisTest = newToThisTest;
```

Comment:
I think this was only used as part of veil functionality, so it should be safe to remove if we removed veil.

Reply:
I think this was only used as part of veil functionality, so it should be safe to remove if we removed veil.

Reply:
Thanks @dapperdrop , that make sense.

Reply:
I was leaning towards not including this because we can infer this from the existance of `decisionIdx` or `seed/hash` on the test object.

It looks clearer having a name `newToTest` on the test object though. So if users only want to fire the tracking once per test, that would save data.

Up to @dapperdrop whether we keep this or not.

Reply:
I've re-instated this in: https://github.com/mint-metrics/mojito-js-delivery/pull/37/commits/f20f1f334a85d87ac8e0c0a212cb14c88776cbd9